package/CHANGELOG

```diff
@@ -13,6 +13,17 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+
+09/16/22 aya9aladdin
```

Comment:
Have another look at how we do the CHANGELOG. Here you created a new version block while you only need to update the existing one.

Reply:
I've modified it, sorry for that